test.js

```diff
@@ -32,7 +32,8 @@ test('main', t => {
 	spinner.enabled = true;
 
 	readOutput(spinner, (output) => {
-		t.is(output, '⠋ foo');
+		let expectedOutput = process.platform === 'win32' ? '- foo' : '⠋ foo';
```

Comment:
Just put it in a variable at the top: `const spinnerChar = process.platform === 'win32' ? '-' : '⠋';` and use that in both tests.